.pullapprove.yml

```diff
@@ -12,3 +12,8 @@ reviewers:
   - PromoFaux
   name: pullapprove
   required: 3
+always_pending:
```

Comment:
Any PR starting with WIP|wip is automatically on hold pending and will not clear until renamed.